src/rules/noAnyRule.ts

```diff
@@ -42,7 +43,10 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class NoAnyWalker extends Lint.RuleWalker {
     public visitAnyKeyword(node: ts.Node) {
-        this.addFailureAtNode(node, Rule.FAILURE_STRING);
+        const fix = this.createFix(
+            this.createReplacement(node.getStart(), node.getWidth(), "{}"),
```

Comment:
If you have the `no-any` rule turned on (none of our projects do @nchen63), this seems like a reasonable fix to me. Auto-fixes can always be reverted, so I'm open to trying this out and waiting for feedback.